src/main/java/com/example/springsample/SeatReservationService.java

```diff
@@ -0,0 +1,26 @@
+package com.example.springsample;
+
+import org.springframework.stereotype.Service;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Service
+public class SeatReservationService {
+
+    // This map stores the reservations with movieId as key and seatNumber as value
+    private final Map<String, Integer> reservations = new HashMap<>();
```

Comment:
**Suggestion:** Consider adding synchronization or using a concurrent map implementation to ensure thread safety, as multiple threads might access the `reservations` map concurrently. [security, importance: 10]
```suggestion
    private final Map<String, Integer> reservations = new ConcurrentHashMap<>();
```